blazor/pager/responsive-layout.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Responsive Layout in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about responsive layout in the Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Responsive Layout in Pager Component
+
+## Responsive
+
+The Pager component has responsive support such that component also fit with mobile resolutions. While resizing the browser window, the inner elements in the Pager component will adjust automatically to equalize the size.
+
+![Blazor Pager with Responsive](./images/blazor-pager-responsive-layout.gif)
```

Comment:
it is not correct to show the same gif image of grid. I asked to put as an image instead of gif. It is enough to take one image to show the responsive layout of pager